Flixter/DetailsViewController.h

```diff
@@ -11,6 +11,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface DetailsViewController : UIViewController
 
+@property (nonatomic, strong) NSDictionary *movieDetails;
```

Comment:
This is code path's fault but they are so wrong to teach this. Let's talk tomorrow about what this should be instead.